docs/content/node-operation/node-setup.mdx

```diff
@@ -131,16 +131,210 @@ Replace `$ROLE` with the node type you are planning to run. Valid options are:
   ctr images pull gcr.io/flow-container-registry/${ROLE}:alpha-v0.0.1",
 ```
 
-## Set Your Node to Start
+## Prepare Your Node to Start
 
-Your nodes will need to boot at startup, and come back to life if they crash.
+Your nodes will need to boot at startup, and restart if they crash. 
 
-If you are running `systemd` you can use the service files provided by `flow-go`. Find them in the [Flow Go](https://github.com/onflow/flow-go/tree/master/deploy)
+If you are running `systemd` you can use the service files provided by `flow-go`. 
+Find them in the [Flow Go](https://github.com/onflow/flow-go/tree/master/deploy). 
+
+If you are using some other system besides Systemd, you need to ensure that the Flow container is started,
+the appropriate key directories are mounted into the container, and that the container will
+automatically restart following a crash.
+
+The `systemd` files pull runtime setting from `/etc/flow/runtime-config.env` and any `.env`
+files under `/etc/flow/conf.d`. Examples of these files are also available in the github repo.
+You will need to modify the runtime config file later.
+
+### Systemd
+
+<Callout type="info">
+If you are not using Systemd, you can skip this step
+</Callout>
+
+1. Ensure that you pulled the latest changes from [flow-go repository](https://github.com/onflow/flow-go) via `git`
+
+```shell
+## Clone the repo if you haven't already done so
+git clone https://github.com/onflow/flow-go
+
+## Get latest changes
+cd flow-go
+git pull origin master
+```
+
+2. Copy your respective [systemd unit file](https://github.com/onflow/flow-go/tree/master/deploy/systemd-docker) to: `/etc/systemd/system`
+3. Create directory `sudo mkdir /etc/flow`
+4. Copy the [runtime-conf.env](https://github.com/onflow/flow-go/blob/master/deploy/systemd-docker/runtime-conf.env) file to: `/etc/flow/`
+5. Enable your service `sudo systemctl enable YOUR-FLOW-NODE.service`
```

Comment:
Is YOUR-FLOW-NODE the node ID or some arbitrary string?

Reply:
It's `flow-$ROLE`, eg: `flow-collection.service`